pkg/apis/minio.min.io/v2/types.go

```diff
@@ -278,6 +278,13 @@ type TenantSpec struct {
 	PrometheusOperator bool `json:"prometheusOperator,omitempty"`
 	// *Optional* +
 	//
+	// The name of the Prometheus instance to scrape metrics from.
```

Comment:
Same as above. Prometheus scrapes metrics from underlying system and here rather are listing API end points to scrape metrics from.